scripts/analyze.sh

```diff
@@ -175,6 +175,20 @@ if check_for_require aio || check_config_file ZJS_AIO; then
     echo "export ZJS_AIO=y" >> zjs.conf.tmp
 fi
 
+if check_for_require fs || check_config_file ZJS_FS; then
+    >&2 echo Using module: FS
+    MODULES+=" -DBUILD_MODULE_FS -DBUILD_MODULE_BUFFER"
+    echo "CONFIG_FILE_SYSTEM=y" >> prj.conf.tmp
+    echo "CONFIG_FILE_SYSTEM_FAT=y" >> prj.conf.tmp
+    echo "CONFIG_DISK_ACCESS_FLASH=y" >> prj.conf.tmp
+    echo "CONFIG_FS_FAT_FLASH_DISK_W25QXXDV=y" >> prj.conf.tmp
+    echo "CONFIG_FLASH=y" >> prj.conf.tmp
+    echo "CONFIG_SPI=y" >> prj.conf.tmp
+    echo "CONFIG_GPIO=y" >> prj.conf.tmp
```

Comment:
Yep, unfortunately its not direct memory access. Not sure why it needs GPIO (maybe to turn Flash on) but their example requires it.